learningobjects/tests/test_api.py

```diff
@@ -0,0 +1,62 @@
+"""
+Test api functions.
+"""
+
+from django.test.testcases import TestCase
+
+from django.contrib.auth.models import User
+
+from learningobjects.models import Course
+from learningobjects.api import create_course
+
+# pylint: disable=no-member
```

Comment:
I'm fine with that, then just add
```ini
[MESSAGES CONTROL]

disable = no-member
```
to the pylintrc file and remove all the per file ones.